SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityTaskReconciliation.java

```diff
@@ -44,6 +44,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(
     SingularityTaskReconciliation.class
   );
+  private static final double NEARLY_FULL = 0.9;
```

Comment:
is this a good number to consider nearly full?

Reply:
I think we could always make this a field in SingularityConfiguration so we can edit it later if needed. As a note on size though, I think I'd go slightly lower maybe? Especially for something with 1000s of tasks, we'd want a good amount of overhead space. Maybe more like .75 or .8?

Reply:
oh good point to put it in the SingularityConfiguration -- I'll move it. And yeah, that makes sense to lower it. when we discussed as a team, it sounded like we were already running at 80%, or was I misunderstanding?

Reply:
Added statusQueueNearlyFull value to SingularityConfiguration and set it to 0.8 for now.